metricbeat/modules.d/audit.yml.disabled

```diff
@@ -0,0 +1,3 @@
+- module: audit
+  metricsets: ["kernel"]
```

Comment:
It seems it didn't have the `enabled: false` before, this makes me think I can remove `enabled` from all of them, so we don't have 2 visible ways to disable (rename file vs changing field)